manifest-dev.yaml

```diff
@@ -0,0 +1,10 @@
+---
+applications:
+- name: funding-service-design-assessment-store-dev
+  memory: 64M
+  buildpacks:
+  - https://github.com/cloudfoundry/python-buildpack.git
+  command: gunicorn wsgi:app -c run/gunicorn/devtest.py
+  env:
+    FLASK_ENV: dev
+    FLASK_DEBUG: 1
```

Comment:
Can we add a service to bind to `logit-ssl-drain` for both dev and test? This will make logs propagate through to logit.